### PR TITLE
Initial gui

### PR DESCRIPTION
**Pull request by micoli**

### CODE REVIEW

File: assets/views/DashboardView/Containers.jsx [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+              </TableBody>
+            </Table>
+          </Box>
+        </PerfectScrollbar>
+      </Card>
+  );
+};
+
+Containers.propTypes = {
+  className: PropTypes.string
```
Comment (MarcoHsrv) [inline assets/views/DashboardView/Containers.jsx:77]: Maybe you can add isRequired or add a Containers.defaultProps with a default value for the className

File: assets/views/DashboardView/Examples/Budget.jsx [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+            Since last month
+          </Typography>
+        </Box>
+      </CardContent>
+    </Card>
+  );
+};
+
+Budget.propTypes = {
+  className: PropTypes.string
```
Comment (MarcoHsrv) [inline assets/views/DashboardView/Examples/Budget.jsx:95]: The same, Maybe you can add isRequired or add a Containers.defaultProps with a default value for the className

File: assets/views/DashboardView/Examples/LatestProducts.jsx [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+        >
+          View all
+        </Button>
+      </Box>
+    </Card>
+  );
+};
+
+LatestProducts.propTypes = {
+  className: PropTypes.string
```
Comment (MarcoHsrv) [inline assets/views/DashboardView/Examples/LatestProducts.jsx:125]: One pore time : Maybe you can add isRequired or add a Containers.defaultProps with a default value for the className

File: assets/views/DashboardView/Examples/Sales.jsx [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+        >
+          Overview
+        </Button>
+      </Box>
+    </Card>
+  );
+};
+
+Sales.propTypes = {
+  className: PropTypes.string
```
Comment (MarcoHsrv) [inline assets/views/DashboardView/Examples/Sales.jsx:147]: same: Maybe you can add isRequired or add a Containers.defaultProps with a default value for the className

File: assets/views/DashboardView/index.jsx [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+const Dashboard = () => {
+  const classes = useStyles();
+  return (
+    <Page
+      className={classes.root}
+      title="Dashboard"
+    >
+      <Container maxWidth={false}>
+        <Grid container spacing={3}>
+          <Grid item lg={12} md={12} xl={12} xs={12}>
```
Comment (MarcoHsrv) [inline assets/views/DashboardView/index.jsx:40]: You don't need to define all format when they are the same, you can just define xs

File: assets/views/DashboardView/index.jsx [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+  </Grid>
+  <Grid item lg={3} sm={6} xl={3} xs={12}>
+    <TotalCustomers/></Grid>
+  <Grid item lg={3} sm={6} xl={3} xs={12}>
+    <TasksProgress/>
+  </Grid>
+  <Grid item lg={3} sm={6} xl={3} xs={12}>
+    <TotalProfit/>
+  </Grid>
+*/
```
Comment (MarcoHsrv) [inline assets/views/DashboardView/index.jsx:29]: Useful ?

**Review by MarcoHsrv — COMMENTED** [review]
For the Classname, I don't repeat each time ;)

File: .editorconfig [hunk 7]
```diff
@@ -0,0 +1,13 @@
+root = true
+
+[*.js]
+end_of_line = lf
+insert_final_newline = true
+charset = utf-8
+
+[*.js]
+indent_style = space
+indent_size = 2
```
Comment (AdrienCha) [inline .editorconfig:10]:
```suggestion
indent_size = 4
```

C'est une préférence personnelle ^^

File: .prettierrc [hunk 8]
```diff
@@ -0,0 +1,9 @@
+{
+  "bracketSpacing": true,
+  "printWidth": 80,
+  "singleQuote": true,
+  "trailingComma": "none",
+  "tabWidth": 2,
```
Comment (AdrienCha) [inline .prettierrc:6]:
```suggestion
  "tabWidth": 4,
```

File: Vagrantfile [hunk 9]
```diff
@@ -51,11 +51,11 @@ Vagrant.configure("2") do |config|
 #             chmod o+w /etc/ansible/roles
 #         EOM
 #
-        subconfig.vm.provision "ansible_local" do |ansible|
+        subconfig.vm.provision "ansible" do |ansible|
             ansible.raw_arguments = Shellwords.shellsplit(ENV["ANSIBLE_ARGS"]) if ENV["ANSIBLE_ARGS"]
             ansible.galaxy_role_file = 'provisioning/requirements.yml'
             ansible.playbook = "provisioning/vagrant-slave.yml"
-#             ansible.verbose = 'vvvv'
+            #ansible.verbose = 'vvvv'
```
Comment (AdrienCha) [inline Vagrantfile:58]: Pourquoi commenter plutot que supprimer ?

File: src/Core/ServiceCloner/Exception/NonExistingServiceInstanceException.php [hunk 10]
```diff
@@ -0,0 +1,16 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Core\ServiceCloner\Exception;
+
+use DomainException;
+use Throwable;
+
+class NonExistingServiceInstanceException extends DomainException
```
Comment (AdrienCha) [inline src/Core/ServiceCloner/Exception/NonExistingServiceInstanceException.php:10]:
```suggestion
final class NonExistingServiceInstanceException extends DomainException
```

**Pull request merged**